.circleci/do-rtl-build.sh

```diff
@@ -58,12 +56,19 @@ run "export RISCV=\"$TOOLS_DIR\"; \
      export PATH=\"$REMOTE_VERILATOR_DIR/bin:\$PATH\"; \
      export VERILATOR_ROOT=\"$REMOTE_VERILATOR_DIR\"; \
      export COURSIER_CACHE=\"$REMOTE_WORK_DIR/.coursier-cache\"; \
-     make -C $REMOTE_SIM_DIR clean; \
-     make -j$REMOTE_MAKE_NPROC -C $REMOTE_SIM_DIR FIRRTL_LOGLEVEL=info JAVA_ARGS=\"$REMOTE_JAVA_ARGS\" ${mapping[$1]}"
-run "rm -rf $REMOTE_CHIPYARD_DIR/project"
-
-# copy back the final build
-
+     make -C $REMOTE_SIM_DIR clean;"
```

Comment:
Why do we need a bunch of exports before the clean?  Also, why remove the `rm -rf project` line?

Reply:
Nevermind on the `rm -rf project` comment... it seems like there was some duplication?

Reply:
Technically we just need to export `RISCV` before the clean.

`rm -rf project` was duplicated, I removed one of them.